### ISSUE TITLE
Has the similar url changed?

### ISSUE THREAD

**Issue by analyseDev** [facundoolano/google-play-scraper#304]
* Operating System: N/A
* Node version: N/A
* google-play-scraper version: v6.2.3

### Description: 
It seems that the similar url has changed and a call responds with an error '404 Not Found', so similar.js module doesn't work.

### Example code:
```js
const options = Object.assign({
      url: `https://play.google.com/store/apps/similar?id=${appId}&hl=${opts.lang}`,
      followAllRedirects: true
    }, opts.requestOptions);
```

### Error message:
```
Status: 404 Not Found
```

**Comment by vitaliy901** [comment]
don't work too :(

**Comment by hunnapikes** [comment]
https://github.com/facundoolano/google-play-scraper/pull/303

**Comment by facundoolano** [comment]
Just merged #303 can someone pull from master and confirm it's working as expected?
